Casamari/MNC019.xml

```diff
@@ -474,7 +474,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <extent>
                            <measure unit="leaf">144</measure>
                            <measure unit="leaf" type="blank">11</measure>
-                           <locus target="#1r #1v #2r #2v #3r #3v #4v #5r #5v #6r #6v"/>
+                           <measure unit="page" type="blank">There are 12 protective pages (<locus from="1r" to="3v"/>, <locus from="4v" to="6v"/>, <locus target="#144v"/>).</measure>
```

Comment:
Also f. 4r is a protective page but not blank. 
As I understood, we differ between blank leaves within the textblock and protective leaves (whether or not they are blank). Probably you made theses changes before our disussion... Sorry!

Reply:
Mmh.. actually I followed here one of Pietro's suggestions in https://github.com/BetaMasaheft/Documentation/issues/1708. Perhaps I should add one more "blank" next to "protective". How do you suggest to build up the wording?

Reply:
If I remember it correctly what we discussed together in our last meeting, then we decided the following: We differentiate between
a) blank leaves within the textblock and
b) protective leaves before and after text block (whether they are blank or written or with drawings). 
For protective leaves we do not state if they are blank (since it is difficult to find a consens concerning te question "what is blank". We argued that the general statement on blank leaves does not add a substantial infoamtionon the Ms. Whereas we found the presence of protective leaves and blank leaves within the text block meaningful.

PS: I just called Anto to confirm my recall and she agrees,

Reply:
Sorry, I missed a crucial information then. I believed we were also marking the _blank_ + protective leaves.
But then (I am sorry again) I think I have some questions/observations:
- are protective leaves and guardleaves (those we indicate with `n="A"` and `n="B"`) the same thing? If so, and if we do not mind whether those protective leaves are blank or not, then I would simply omit to state explicitly this information, because it is already contained (or retrievable for PDF) in the full quire list below (recognizable with `n="A"` and `n="B"`).
- Sometimes a manuscript ends up with no final guardleaves (e.g., it only consists of a number of 8-leaf quires), but the last page is blank: is it textblock or protective leaf? If protective leaf = guardleaf, then that last blank page belongs to the textblock (although it is at the end of the textblock) and has to be counted as such. If "protective leaf" is not equal to "guardleaf", then what's the definition?
 
Sorry again to reopen the issue, but I am realizing that at present there are inconsistent statements in my files.  
@SusanneHummel @antobrita

Reply:
You are absolutely right! I also had your last point already in mind. Let's discuss it again. I will add it to our file. Maybe it would be the easiest to keep your data as we als would need to update our files concerning this issue anyway.

Reply:
I didn't merge it in case you would like to take a note or something.

Reply:
Ok, thank you. I have just updated the file with the requested changes.
You can merge the branch, if you want: I've saved our comments above for further discussion.